src/lm_buddy/integrations/huggingface/tokenizer_config.py

```diff
@@ -1,17 +1,10 @@
-from pydantic import field_validator
-
-from lm_buddy.integrations.huggingface import HuggingFaceRepoConfig, convert_string_to_repo_config
-from lm_buddy.integrations.wandb import WandbArtifactConfig
+from lm_buddy.paths import LoadableAssetPath
 from lm_buddy.types import BaseLMBuddyConfig
 
 
 class AutoTokenizerConfig(BaseLMBuddyConfig):
     """Settings passed to a HuggingFace AutoTokenizer instantiation."""
 
-    load_from: HuggingFaceRepoConfig | WandbArtifactConfig
+    load_from: LoadableAssetPath
```

Comment:
maybe also just `AssetPath`? or `ModelPath`

Reply:
It is more general than `ModelPath` since it applies to datasets and tokenizers, so I will call it `AssetPath`.